docker/README.md

```diff
@@ -113,11 +113,13 @@ documentation before completing your upgrade.
 Voxel51 recommends upgrading your deployment using
 [`legacy` authentication mode](https://docs.voxel51.com/teams/pluggable_auth.html#legacy-mode)
 and
-[migrating](https://docs.voxel51.com/teams/pluggable_auth.html#migrating-from-legacy-to-internal-mode)
+migrating
```

Comment:
Line 115-117 should be changed to 


```
and migrating to
```

GitHub won't let me make a suggestion on a deleted line